sdk-lib/src/main/java/cash/z/ecc/android/sdk/model/TransactionOverview.kt

```diff
@@ -5,8 +5,8 @@ package cash.z.ecc.android.sdk.model
  *
  * Note that both sent and received transactions will have a positive net value.  Consumers of this class must
  */
-data class TransactionOverview(
-    val id: Long,
+data class TransactionOverview internal constructor(
```

Comment:
I made this constructor internal because consumers of the SDK probably shouldn't be constructing these objects directly.